src/krfmesg/linux/krfmesg.c

```diff
@@ -17,7 +18,7 @@ int open_netlink(void) {
 
   sock = socket(AF_NETLINK, SOCK_RAW, NETLINK_KRF);
   if (sock < 0) {
-    printf("Failed to make socket. Is krf module installed?\n");
+    err(1, "Failed to make socket. Is krf module installed?");
```

Comment:
Nit: This will actually get formatted strangely since `err` uses the format `program: message: strerror` internally, so this will become something like:

```
krfmesg: Failed to make socket. Is krf module installed?: Access denied
```

The "right" way to format with `err` is something like this:

```c
err(errno, "socket")
```